connector/kafka-0-10-token-provider/src/main/scala/org/apache/spark/kafka010/KafkaConfigUpdater.scala

```diff
@@ -30,7 +30,7 @@ import org.apache.spark.internal.Logging
 /**
  * Class to conveniently update Kafka config params, while logging the changes
  */
-private[spark] case class KafkaConfigUpdater(module: String, kafkaParams: Map[String, Object])
```

Comment:
4.0 added one new internal class in this package: `KafkaTokenProviderExceptions`. Given that this package contains only internal classes, I marked the entire `org.apache.spark.kafka010` package as private, so that we no longer need `private[spark]` in these existing classes.